package.json

```
@@ -1,64 +1,68 @@
-		"@solid-primitives/marker": "^0.1.0",
-		"@solid-primitives/media": "^2.2.9",
-		"@solidjs/meta": "^0.29.4",
-		"@solidjs/router": "^0.15.3",
-		"@solidjs/start": "^1.1.1",
-		"@vinxi/plugin-mdx": "^3.7.2",
-		"dotenv": "^16.4.5",
-		"glob": "^10.4.5",
-		"gray-matter": "^4.0.3",
-		"postcss": "^8.4.47",
-		"shiki": "^1.17.6",
-		"sitemap": "^7.1.2",
-		"solid-heroicons": "^3.2.4",
-		"solid-js": "^1.9.5",
-		"solid-list": "^0.3.0",
-		"solid-mdx": "^0.0.7",
-		"vinxi": "^0.5.3",
-		"zod": "^3.23.8"
-	},
-	"devDependencies": {
-		"@kobalte/tailwindcss": "^0.9.0",
-		"@orama/crawly": "^0.0.4",
-		"@tailwindcss/typography": "^0.5.15",
-		"@types/node": "^22.9.0",
-		"@typescript-eslint/eslint-plugin": "^8.7.0",
-		"@typescript-eslint/parser": "^8.7.0",
-		"autoprefixer": "^10.4.20",
-		"eslint": "^8.57.1",
-		"eslint-plugin-solid": "^0.13.2",
-		"prettier": "3.2.5",
-		"tailwindcss": "^3.4.11",
-		"typescript": "^5.6.2",
-		"vite": "^6.1.1"
-	},
-	"engines": {
-		"node": ">=18",
-		"packageManager": "pnpm@8"
-	}
-}
+{
+	"name": "solid-docs-next",
+	"version": "1.0.0",
+	"private": true,
+	"description": "Docs for Solid",
+	"type": "module",
+	"scripts": {
+		"sync": "node scripts/collections/index.mjs",
+		"sync:orama": "pnpm build && node scripts/sync-orama.mjs",
+		"dev": "pnpm sync && vinxi dev",
+		"build": "pnpm sync && pnpm build:sitemap && vinxi build",
+		"build:sitemap": "node scripts/generateSitemap.js",
+		"start": "vinxi start",
+		"e2e": "npx playwright test",
+		"check:lint": "pnpm dlx eslint",
+		"check:types": "tsc --noEmit",
+		"check:format": "prettier . --check",
+		"format": "prettier . --write"
+	},
+	"dependencies": {
+		"@kobalte/core": "^0.13.9",
+		"@kobalte/solidbase": "^0.0.23",
+		"@lunariajs/core": "^0.0.31",
+		"@oramacloud/client": "^1.3.16",
+		"@solid-primitives/event-listener": "^2.3.3",
+		"@solid-primitives/marker": "^0.1.0",
+		"@solid-primitives/media": "^2.2.9",
+		"@solidjs/meta": "^0.29.4",
+		"@solidjs/router": "^0.15.3",
+		"@solidjs/start": "^1.1.1",
+		"@vinxi/plugin-mdx": "^3.7.2",
+		"dotenv": "^16.4.5",
+		"glob": "^10.4.5",
+		"gray-matter": "^4.0.3",
+		"postcss": "^8.5.3",
+		"shiki": "^1.22.0",
+		"sitemap": "^7.1.2",
+		"solid-heroicons": "^3.2.4",
+		"solid-js": "^1.9.5",
+		"solid-list": "^0.3.0",
+		"solid-mdx": "^0.0.7",
+		"vinxi": "^0.5.3",
+		"zod": "^3.23.8"
+	},
+	"devDependencies": {
+		"@eslint/eslintrc": "^3.3.0",
+		"@eslint/js": "^9.22.0",
+		"@kobalte/tailwindcss": "^0.9.0",
+		"@orama/crawly": "^0.0.4",
+		"@tailwindcss/typography": "^0.5.15",
+		"@types/node": "^22.9.0",
+		"@typescript-eslint/eslint-plugin": "^8.26.0",
+		"@typescript-eslint/parser": "^8.26.0",
+		"autoprefixer": "^10.4.20",
+		"eslint": "^9.22.0",
+		"eslint-plugin-solid": "^0.14.5",
+		"globals": "^16.0.0",
+		"prettier": "3.2.5",
+		"tailwindcss": "^3.4.14",
+		"typescript": "^5.8.2",
+		"vite": "^6.2.1",
+		"vite-plugin-eslint": "^1.8.1"
+	},
+	"engines": {
+		"node": ">=18",
+		"pnpm": ">=10"
+	}

```

```suggestion
	"engines": {
		"node": ">=18",
		"pnpm": ">=9"
	}
```

The lockfile is generated using pnpm v9 and the CI steps also use v9 which conflicts with this.